torch/ao/quantization/fx/_decomposed.py

```diff
@@ -3,6 +3,7 @@
 from typing import Optional
```

Comment:
we are deprecating these, can you add to torchao? or maybe it's already supported in torchao

Reply:
Yeah. Looks like there isn't such an issue with the ops in Torchao.